static/sass/_snapcraft_market.scss

```diff
@@ -116,4 +118,82 @@
       line-height: 2.3rem;
     }
   }
+
+  .p-market-banner {
+
+    .p-market-banner__image-holder {
+      cursor: pointer;
+      position: relative;
+    }
+
+    .p-market-banner__image {
+      background-color: $color-x-light;
+      background-position: center;
+      background-size: cover;
+      border: 1px solid $color-mid-light;
+      border-radius: $border-radius;
+      color: $color-link;
+      cursor: pointer;
+      height: 128px;
+      margin-bottom: $spv-inter--regular;
+      position: relative;
+      text-align: center;
+    }
+
+    .p-market-banner__image-holder:hover,
+    .p-market-banner__image-holder.is-focused {
+      .p-market-banner__image {
+        border: 1px solid $color-link;
+      }
+
+      .p-market-banner__remove {
+        opacity: 1;
+      }
+
+      .p-market-banner__change {
+        border: 1px solid $color-link;
+        border-radius: $border-radius;
+        opacity: 1;
+      }
+    }
+
+    .p-market-banner__remove {
```

Comment:
I guess we have very similar styles in screenshots as well.
Would be good to extract them into some common pattern, so when we want to change them we can do it consistently from one place.